dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -250,6 +256,8 @@ namespace dwa_local_planner {
     return latched_inner_goal_ ||  bypassed_goal || oscillating_;
   }
 
+template <typename T> inline T sign(const T x) { return x<T(0) ? T(-1):T(1); };
```

Comment:
S: could use [std::signbit](https://en.cppreference.com/w/cpp/numeric/math/signbit) for this